packages/plugin-e2e/src/auth/auth.setup.ts

```diff
@@ -1,7 +1,7 @@
 import { test as setup } from '../';
 
 setup('authenticate', async ({ login, createUser, user }) => {
-  if (user) {
+  if (user && !user.skipCreateUser) {
```

Comment:
Not sure I'm understanding the skipCreateUser. Doesn't createUser [already handle cases](https://github.com/grafana/plugin-tools/pull/930/files#diff-ac3488fe952ba2e1dc2e3f2af971744712ff325fb4fff817d3b55f4b21117834R43) where the user already exists?

Reply:
No that's right, user is only created if it doesn't exist, so this is not entirely necessary. I guess I thought this would make this more clear from an end-user standpoint, but that's not the case so I'll remote this prop. Thanks for feedback.